volunteerapp/models.py

```diff
@@ -33,6 +33,18 @@ class Tag(models.Model):
     def __str__(self):
         return self.name
 
+class ProjectEmail(models.Model):
+    user = models.ForeignKey(User, on_delete=models.CASCADE)
```

Comment:
Please don't have this on_delete statement. We want the record even if the user is deleted.